ext/storage_sources/gcp_store/test/test_gcp_connection.cpp

```diff
@@ -31,4 +31,13 @@
 
 #include "gcp_connection.h"
 
-TEST_CASE("Testing class gcpConnection", "gcp-connection") {}
+TEST_CASE("Testing class gcpConnection", "gcp-connection")
```

Comment:
Let's remove this test case since putObject is not implemented yet.

Reply:
fixed